.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/node:10-browsers
+    steps:
+      - checkout
+      - run: npm config set prefix "$HOME/.local"
+      - run: npm i -g origami-build-tools@^7
+      - run: $HOME/.local/bin/obt install
+      - run: $HOME/.local/bin/obt demo --demo-filter pa11y --suppress-errors
+      - run: $HOME/.local/bin/obt verify
+      - run: $HOME/.local/bin/obt test
```

Comment:
can this use `origami-ci`?

Reply:
Oh yes! This is from `obt init`, I'll open a PR against the boilerplate